### DISCUSSION TITLE
Something in the files.

### DISCUSSION THREAD

**Discussion by CrewieAustin** [Monika-After-Story/MonikaModDev#9496] [QA - tech support + general] [answered]
I've noticed there's a wet hair variant of Monika's hairstyle and a white towel clothing. What are those for?? ._.
![Wet_Hair_Screenshot](https://user-images.githubusercontent.com/86544170/188569752-0933a41a-cbf4-4a60-bd5e-5fdb728937bc.png)
![Bath_Towel_Screenshot](https://user-images.githubusercontent.com/86544170/188570164-9c541b25-561b-4e64-9723-a6c99655fbf7.png)

**Comment by Booplicate** [comment, marked as answer]
Monika does hot tub streams.

**Reply by CrewieAustin** [reply]
Hot Tub Streams? Is it some sort of an event or did I need certain affection points for Monika to bring this conversation up? Sorry for asking too much, I'm just curious to stuff like these.

**Reply by Justformas** [reply]
> I'm just curious

Very understandable. It's not really a conversation she brings up. It's a new Monika room greeting in this version. But greetings are pretty random, so it'll happen when, or if, it happens.
